recipes/redis/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = '3.2.0' %}
+
+package:
+  name: redis
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/antirez/redis.git
+  git_rev: "{{ version }}"
+
+build:
+  number: 0
+  skip: true  # [win]
+
+test:
+  requires:
+    - tk 8.5*
+  commands:
+    - "cd {{ SRC_DIR }} && tclsh8.5 tests/test_helper.tcl"
+
+about:
+  home: http://redis.io
+  license: BSD
```

Comment:
3.  Should this field's value be `BSD 3-Clause`, `3-Clause BSD`, or something else?